### DISCUSSION TITLE
Welcome to alphatrade Discussions!

### DISCUSSION THREAD

**Discussion by algo2t** [algo2t/alphatrade#3] [Q&A]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, guys github is giving good platform to ask question. Instead of creating issue you can ask questions and get answers for them here.

**Comment by vinay432100** [comment]
Hi Mate,
Your work is very awesome. I truly appreciate that.
Struggling how to get accesstoken.txt flie.
Should I mail for that to SAS Online Trading.
Thank you in advance.
